<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu_edge_collapse</class>
 <widget class="QDialog" name="Menu_edge_collapse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>487</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>99999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edge collapse</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="txt_nb_vertices">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Number of collapses</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_collapse">
     <property name="text">
      <string>Perform collapse</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Number of vertices after collapse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Number of vertices</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QSlider" name="horizontalSlider">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Policy</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="m_simplificationMethod">
        <item>
         <property name="text">
          <string>Structure preserving (memoryless)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Structure preserving</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Garland Heckbert</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Min length and midpoint</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lindstrom Turk</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="spinBox_nb_remaining_vertices">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="txt_nb_collapses">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
     <property name="title">
      <string>Simplification parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Abstraction weight</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_abstraction_weight">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Border weight</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_boundary_weight">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>0.800000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Valid criterions</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="checkBox_preserve_normals">
        <property name="text">
         <string>Preserve normals angle</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_preserve_normals">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>150.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QCheckBox" name="checkBox_link_condition">
        <property name="text">
         <string>Link condition</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
